### ISSUE TITLE
Run example notebooks as tests automatically

### ISSUE THREAD

**Issue by ranocha** [ketch/nodepy#87]
Is there a way to test all example notebooks automatically? This would have prevented the recent issues #83, #84, #85, #86.

**Comment by ketch** [comment]
Yes. I have set that up for other projects and will do the same here.
